MAINTAINERS

```diff
@@ -0,0 +1,4 @@
+Adrian Reber <areber@redhat.com>
+Andrey Vagin <avagin@gmail.com>
```

Comment:
pls remove me from maintainers. I am always ready to help but I don't involve enough into this project.

Reply:
Sure, no worries. I have updated the pull request.